terraform/container.go

```diff
@@ -291,6 +288,53 @@ func (terraformContainer *Container) listWorkspaces(errorStream io.Writer) (map[
 	return result, nil
 }
 
+func (terraformContainer *Container) CopyTerraformLockIfExists(outputStream, errorStream io.Writer) error {
+	lockExists, err := terraformContainer.CheckFileExists("/build/.terraform.lock.hcl", errorStream)
+	if err != nil {
+		return err
+	}
+
+	if lockExists {
+		fmt.Fprintf(
+			errorStream,
+			"\n%s\n%s\n",
+			util.FormatInfo("copying .terraform.lock.hcl from release"),
+			util.FormatCommand("cp /build/.terraform.lock.hcl /code/infra/"),
+		)
+
+		if err := terraformContainer.RunCommand([]string{"cp", "/build/.terraform.lock.hcl", "/code/infra/"}, map[string]string{}, outputStream, errorStream); err != nil {
+			return err
+		}
+	} else {
+		fmt.Fprintf(
+			errorStream,
+			"\n%s\n%s\n",
+			util.FormatInfo(".terraform.lock.hcl not found"),
+			util.FormatCommand("cp /build/.terraform.lock.hcl /code/infra/"),
+		)
+	}
+
+	return nil
+}
+
+func (terraformContainer *Container) CheckFileExists(path string, errorStream io.Writer) (bool, error) {
+	var outputBuffer bytes.Buffer
+	command := fmt.Sprintf("test -f %s && echo exists || echo none", path)
```

Comment:
The "none" string is never used. I guess it's useful to have to || in order to make the command always successful. Could also be `test -f %s && echo exists || true`, although doesn't make a lot of difference.

Reply:
true I think is confusing it could be false, what do you think?